Dockerfile

```diff
@@ -1,13 +1,20 @@
 FROM node:lts-slim  AS runner
 
 RUN apt-get update && \
-    apt-get install -y jq && \
+    apt-get install -y --no-install-recommends jq opam ca-certificates build-essential && \
```

Comment:
Would `build-essentials` significantly increase the image size?

Reply:
Unfortunately it seems we need it to properly install ocaml on the image

Reply:
Can it be removed once ocaml is installed/set up or is it needed at runtime? The ca-certs shouldn't be needed to run tests since the tests shouldn't be talking to the network.

Reply:
Turns out `build-essential` is not needed. Removed it.

Will figure out the rest